### PR TITLE
Ccp 2202

### PR DESCRIPTION
**Pull request by Carsten89**

### CODE REVIEW

File: frontend/subscriptions.js [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+  // "live chat" -> userlike-tab. "UM" -> #uslk-button
+  css.global('#userlike-tab, #uslk-button', {
+    bottom: 'calc(10px + var(--safe-area-inset-bottom) + var(--footer-height) ) !important',
+  });
+  css.global('#userlikeButtonContainer', {
+    display: 'var(--userlike-um-display) !important',
+  });
+
+  css.global('#userlike-popup', {
+    top: 'var(--safe-area-inset-top) !important',
```
Comment (alexbridge) [inline frontend/subscriptions.js:23]: I  guess popup should also respect a `--safe-area-inset-bottom`

File: frontend/subscriptions.js [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+import { sdkUrl, pagesWithoutWidget } from './config';
+import { getUserData } from './selectors';
+
+export default (subscribe) => {
+  const { style } = document.documentElement;
+  let ready = false;
+
+  // "live chat" -> userlike-tab. "UM" -> #uslk-button
+  css.global('#userlike-tab, #uslk-button', {
+    bottom: 'calc(10px + var(--safe-area-inset-bottom) + var(--footer-height) ) !important',
```
Comment (alexbridge) [inline frontend/subscriptions.js:16]: As you see on video, it changes position on PDP when Sheet comes up, because of footer height

**Review by alexbridge — CHANGES_REQUESTED** [review]

File: frontend/subscriptions.js (hunk 1 above)
Comment (Carsten89) [reply frontend/subscriptions.js:23]: done

File: frontend/subscriptions.js (hunk 2 above)
Comment (Carsten89) [reply frontend/subscriptions.js:16]: done

**Pull request merged**